### DISCUSSION TITLE
Meeting schedule adjustment

### DISCUSSION THREAD

**Discussion by jongfeel** [Joseph-Cha/JustFight_Client#56] [Ideas]
영어 공부를 하신다고 하여 시간을 조절한다는 얘기를 했었습니다.

그래서 아래와 같은 일정으로 조절해 보고, 프로젝트 범위도 한번 조절해 보면 어떨까 합니다.

- 미팅 주기
  - 기존: 주1회 화
  - 현재: 주1회가 잘 진행되지 않고 월,화 등 고정적이지 않음
  - 변경: 주2회 일요일 오후 11시 제안
- 투입 시간 조절
  - 기존: ??
  - 변경: ??
- 프로젝트 범위 조절
  - 기존: 1년의 기간, 그리고 게임의 완성
  - 변경: 지난 6개월간 진행한 만큼 보다 더 적은 범위로 산정.

**Comment by Joseph-Cha** [comment]
월 2회 일요일 오후 11시 정도로 생각하신거죠??
좋은 것 같아요!
그리고 투입 시간은 주 2시간 정도로 잡아야할 것 같습니다

프로젝트 범위 조정 및 멘토링 미팅은 
요번주에는 이사 정리하느라 정신이 없을 것 같아서 다음 주 일요일에 하면 좋을 것 같습니다 ㅎ
